### PR TITLE
Bah 1332

### PR DESCRIPTION
**Pull request by rohit-yawalkar**

### CODE REVIEW

File: event-log-service-webapp/pom.xml [hunk 1]
```diff
@@ -36,13 +36,11 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-api</artifactId>
             <version>${log4j.version}</version>
-            <scope>provided</scope>
```
Comment (binduak) [inline event-log-service-webapp/pom.xml]: @rohit-yawalkar @deepthi-mantena do we have to package the log4j as part of this jar ? any specific reason for it ?

Comment (rohit-yawalkar) [reply event-log-service-webapp/pom.xml]: yes @binduak, we packaged because we are not using openmrs-api/core over here

Comment (binduak) [reply event-log-service-webapp/pom.xml]: i think thats still fine. Have you encountered any issue for building omod/runtime ?

Comment (rohit-yawalkar) [reply event-log-service-webapp/pom.xml]: No Bindu, its building fine

Comment (binduak) [reply event-log-service-webapp/pom.xml]: in that case let add the scope as provided for now ?

**Review by binduak — APPROVED** [review]

**Pull request merged**